src/config/data/chains.json

```diff
@@ -60,8 +84,22 @@
       "GNOSIS_MAINNET": {
         "chainId": 100,
         "title": "Gnosis Chain mainnet",
-        "explorer": "https://blockscout.com/poa/xdai",
-        "nativeCurrency": { "name": "xDAI", "symbol": "XDAI", "decimals": 18 }
+        "explorer": "https://gnosis.blockscout.com/",
```

Comment:
Can we do `https://gnosisscan.io` here? That's the Etherscan clone and where we're verifying contracts.

Reply:
chains.json is auto-generated from [chainlist](https://chainlist.org/) data. This is used to set the explorer when developer add LINK to their wallets. It shouldn't apply to URLs in our documentation, but we should double check that.

Reply:
I manually updated it in this PR anyway